src/devices/bus/coco/cococart.cpp

```diff
@@ -48,6 +48,18 @@
 #define LOG_LINE                0
 
 
+/***************************************************************************
+	CONSTANTS
+***************************************************************************/
+
+enum
+{
+	TIMER_CART,
+	TIMER_NMI,
+	TIMER_HALT
+};
```

Comment:
cococart_slot_device will never be subclassed.  device_cococart_interface gets subclassed.